### PR TITLE
Ashley ch12

### PR DESCRIPTION
**Pull request by ashleychien**
Made phrasing changes in 12.3
Made two concept changes in 12.3

### CODE REVIEW

File: ch12/linear_multiple.md [hunk 1]
```diff
@@ -1 +1 @@
-!INCLUDE "../notebooks-html/linear_multiple.html"
+!INCLUDE "../notebooks-html\linear_multiple.html"
```
Comment (SamLau95) [inline ch12/linear_multiple.md]: Hmm. It looks like the reverse backslashes `\` should be regular slashes `/`. This looks like a issue with running the conversion scripts on Windows. For now, can you replace the `\` with `/` for this `.md` file?

File: notebooks-html/linear_multiple.html [hunk 2]
```diff
@@ -505,35 +507,36 @@ <h2 id="Multiple-Linear-Regression">Multiple Linear Regression<a class="anchor-l
...
-<p>Notice that $ X $ is composed of multiple $ x $ vectors stacked on top of each other. To keep the notation clear, we define $ X_{i} $ to refer to the row with index $ i $ of $ X $. We define $ X_{ij} $ to refer to the element with index $ j $ of the row with index $ i $ of $ X $. Thus, $ X_i $ is a $ p $-dimensional vector and $ X_{ij} $ is a scalar. $ X $ is an $ n \times p $ matrix.</p>
-<p>For example, from the table above we have $ X_4 = [1, 140, 3449, 70] $ and $ X_{41} = 140 $. This notation becomes important when we define the cost function since we will need both $ X $, the matrix of input values, and $ y $, the vector of MPGs.</p></div></div></div>
+<p>Notice that $ X $ is composed of multiple $ x $ vectors stacked on top of each other. To keep the notation clear, we define $ X_{i} $ to refer to the row with index $ i $ of $ X $. We define $ X_{i,j} $ to refer to the element with index $ j $ of the row with index $ i $ of $ X $. Thus, $ X_i $ is a $ p $-dimensional vector and $ X_{i,j} $ is a scalar. $ X $ is an $ n \times p $ matrix, where $n$ is the number of car examples we have and $p$ is the number of attributes we have for a single car.</p>
```
Comment (SamLau95) [inline notebooks-html/linear_multiple.html]: I like this change! Especially since it resolves the ambiguity when a matrix contains more than 10 rows.

**Comment by SamLau95** [comment]
Can you also pull from master? I have an update to the build script to generate markdown files which will make reviewing a lot easier for me.

**Comment by ashleychien** [comment]
I fixed the slash and also pulled from master! Running `make ch12` is still saying "make: Nothing to be done for 'ch12'." Not sure if this is the expected behavior. Got it about concept changes. Tiff and I haven't really figured out huge concept changes that we wanted to do for section 12.3, so I did some small things  in 12.3 and helped Tiff with some concept changes in her sections (12.1 - 12.3).

**Comment by SamLau95** [comment]
I'll look into the make issue.

**Comment by zhunation** [comment]
I have the same make issue, might be a windows thing? I've been just manually running the commands the make file is supposed to run

**Pull request merged**